tcs-service/Dockerfile

```diff
@@ -4,4 +4,4 @@ EXPOSE 8093
 
 COPY target/app.jar app.jar
 
-CMD ["java", "-jar", "app.jar"]
+CMD ["java", "-XX:MaxRAMPercentage=95", "-jar", "app.jar"]
```

Comment:
Looks good to me. Just wondering, together with this, can we also use garbage collector  "-XX:+UseG1GC" ?

Reply:
We absolutely could look to add additional optimisations into the JVM.  I want to avoid over-polishing work for the small well defined story this satisfies.